### PR TITLE
implemented pipe operator

### PR DESCRIPTION
**Pull request by avranil26**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -72,6 +74,14 @@ To simplify searches, this API also has provided the feature of provided the fun
...
 Ctrl('r') is the way to invoke that functionality.
 
+# General attributes
+
+1. | (pipe operator)
+The above command will combine multiple command and run them sequentially.
+
+Sample Syntax:
+
+listDir -a|listDir -l
```
Comment (ov-wagle) [inline README.md]: Spacing before and after |

File: src/pipe_operator.rs [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+                println!("Invalid command");
+
+            }
+            else{
+
+                for items in chunks.iter(){
+
+                    let cmd = items.trim().to_string();
+                    let history = util::initialize_vector();
```
Comment (ov-wagle) [inline src/pipe_operator.rs:47]: don't initialize a new vector instance here. Instead, take history as the command argument, pass history from util itself and return history from this function, This way, the ownership of variable history is again returned back to the caller

**Review by ov-wagle — CHANGES_REQUESTED** [review]

File: src/pipe_operator.rs (hunk 2 above)
Comment (avranil26) [reply src/pipe_operator.rs:47]: This is a dummy history vector. It does not update the actual history vector in the main. Taking the history from the main will update the vector and would need to remove the last few commands

File: README.md (hunk 1 above)
Comment (avranil26) [reply README.md]: Works both ways. With or without spacing before pipe. But will update it in the REARDME

File: src/pipe_operator.rs (hunk 2 above)
Comment (avranil26) [reply src/pipe_operator.rs:47]: This is also an immutable history vector which is initialized

**Review by ov-wagle — APPROVED** [review]

**Pull request merged**